src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -679,9 +670,10 @@ TEST_CASE("request_rate_streaming: test that streaming-specific logic works")
   {
     TestRequestRateManager trrm(
         params, false /* is_sequence */, true /* is_decoupled */);
-    trrm.schedule_.push_back(nanoseconds(1));
 
     auto worker = trrm.MakeWorker(thread_stat, thread_config);
+    std::dynamic_pointer_cast<IScheduler>(worker)->SetSchedule(
+        {nanoseconds(1)}, nanoseconds(1));
```

Comment:
Might as well stay consistent between this sublease and `enable_stats false` with how these arguments are created and passed.

Reply:
I completely joined the two tests other than the two bools that are different